utils/queries.tsx

```diff
@@ -1,13 +1,29 @@
-import { User, UserStatus } from '../xplat/types/types';
+import { getUserById } from '../xplat/api';
+import {
+  Cursor,
+  Post,
+  RouteClassifier,
+  RouteType,
+  User,
+  UserStatus,
+} from '../xplat/types/types';
 
-type FetchedUser = {
+export interface FetchedUser {
   username: string;
   email: string;
   displayName: string;
   bio: string;
   status: UserStatus;
   avatarUrl: string;
-};
+  followingList: User[];
+  bestBoulder: RouteClassifier | undefined;
+  bestToprope: RouteClassifier | undefined;
+  totalSends: number;
+  postsCursor: Cursor<Post>;
+  followersCursor: Cursor<User>;
+  followingCursor: Cursor<User>;
+  __userObject: User;
```

Comment:
Can we remove the `__` prefix on this? The `userObject` will be used quite often